pkg/graph/resolvers/system_intake_grb_review_form.go

```diff
@@ -100,3 +102,34 @@ func UpdateSystemIntakeGRBReviewFormInputTimeframeAsync(
 		SystemIntake: updatedIntake,
 	}, nil
 }
+
+// PointerToSystemIntakeGRBReviewAsyncStatusType Create a reference to an Async GRB Review status
+func PointerToSystemIntakeGRBReviewAsyncStatusType(
```

Comment:
We have a generic pointer helper method that can do this functionality. I suggest we use that instead
`helpers.PointerTo(status)`